docs/source/asr/intro.rst

```diff
@@ -40,10 +162,27 @@ A demo below allows evaluation of NeMo ASR models in multiple langauges from the
     </script>
 
 
-The full documentation tree is as follows:
+ASR tutorial notebooks
+----------------------
+Hands-on speech recognition tutorial notebooks can be found under `the ASR tutorials folder <https://github.com/NVIDIA/NeMo/tree/main/tutorials/asr>`_.
+If you are a beginner to NeMo, consider trying out the `ASR with NeMo <https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/ASR_with_NeMo.ipynb>`_ tutorial.
+This and most other tutorials can be run on Google Colab by specifying the link to the notebooks' GitHub pages on Colab.
+
+ASR model configuration
+-----------------------
+Documentation regarding the configuration files specific to the ``nemo_asr`` models can be found in the :doc:`Configuration Files <./configs>` section.
+
+Preparing ASR datasets
+----------------------
+NeMo includes preprocessing scripts for several common ASR datasets. The :doc:`Datasets <./datasets>` section contains instructions on
+running those scripts. It also includes guidance for creating your own NeMo-compatible dataset, if you have your own data.
+
+Further information 
+-------------------
+For more information, see additional sections in the ASR docs on the left-hand-side menu or in the list below:
 
 .. toctree::
-   :maxdepth: 8
+   :maxdepth: 1
```

Comment:
What does this reduce ?

Reply:
If you look here https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/intro.html, underneath "The full documentation tree is as follows", it lists the full documentation tree with a lot of nested headings. There is way too much text as a result. I would like to limit it just to the top-level names of the other sections in the asr folder.

Reply:
Oh ok. Makes sense